Makefile

```diff
@@ -151,10 +151,10 @@ docker-clean-worker:
 
 docker-build-worker: docker-clean-worker
 	docker run \
-		-v $(PWD):/go/src/github.com/pachyderm/pachyderm \
+		-v $(PWD):/pachyderm \
```

Comment:
Not opposed to this path change, but just curious about it: do go modules not work under `$GOPATH` or something? Or is this just taking the opportunity to make the path a little nicer?

Reply:
IIRC, I was hitting some issue and wanted to rule out Go's sometimes semantically significant pathing (i.e., at least in Go < 1.13, module-enabled repos could not be in `$GOPATH`.) Even if Go 1.13 removed this limitation, I feel like putting the repo outside of `$GOPATH` reduces ambiguity.